File: PHP/Account.php
<?php
class Account{
    
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    public function __construct($name, $document){

        $this->name = $name;
        $this->document = $document;
    }

    public function PrintDataAccount(){
        echo "name: $this->name, document: {$this->document}";
      }
}

File: Python/account.py
from ast import Str
from xml.dom.minidom import Document


class Account:
    id          = int
    name        = Str
    document    = Str
    email       = Str
    password    = Str

    def __init__(self, name, document):
        self.name = name
        self.document = document
